README.md

```diff
@@ -75,22 +75,6 @@ FIFO allows you the option to configure message deduplication and guarantees ord
 
 To publish a message to a FIFO topic you must include a `MessageGroupId`.
 
-#### SNS Extended Client backed by S3
```

Comment:
hi. at the moment, we use the SNS extended client to manage large messages. removing this will break that feature